docs/user/debugging.md

```diff
@@ -71,28 +83,143 @@ More information can be found in the [View Logs with Logcat][] help article.
+
+Right-click on `Item 0` and select `Add Row` from the context menu.  In the dropdown menu, select `URL Schemes` to add the item.
+
+Click on the disclosure icon of `URL Schemes` to expand the item, double-click the value field of `Item 0` and key in the value for your application's custom scheme.  For instance, the Glean sample app uses `glean-sample-app`, which allows for custom URL's to be crafted using that as a protocol, for example: `glean-sample-app://glean?logPings=true`
+
+#### Add the `Glean.handleCustomUrl()` convenience function and necessary overrides
+
+In order to handle the incoming Glean SDK debug commands, it is necessary to implement the override in the application's `AppDelegate.swift` file.  Within that function, you can make use of the convenience function provided in Glean `handleCustomUrl(url: URL)`.  
+
+An example of a simple implementation of this would look like this:
+
+```Swift
+func application(_: UIApplication,
+                  open url: URL,
+                  options _: [UIApplication.OpenURLOptionsKey: Any] = [:]) -> Bool {
+    // ...
+
+    // This does nothing if the url isn't meant for Glean.
+    Glean.shared.handleCustomUrl(url: url)
+
+    // ...
+
+    return true
+}
+```
+
+If you need additional help setting up a custom URL scheme in your application, please refer to [Apple's documentation](https://developer.apple.com/documentation/uikit/inter-process_communication/allowing_apps_and_websites_to_link_to_your_content/defining_a_custom_url_scheme_for_your_app).
+
+### Invoking the Glean-iOS debug commands
+
+Now that the app has the Glean SDK debug functionality enabled, there are a few ways in which we can invoke the debug commands.
+
+#### Using a web browser
+
+Perhaps the simplest way to invoke the Glean SDK debug functionality is to open a web browser and type/paste the custom URL into the address bar.  This is especially useful on an actual device because there isn't a good way to launch from the command line and process the URL for an actual device.
+
+Using the glean-sample-app as an example: to activate ping logging, tag the pings to go to the Glean Debug View, and force the `events` ping to be sent, enter the following URL in a web browser on the iOS device:
+
+```shell
+glean-sample-app://glean?logPings=true&tagPings=My-ping-tag&sendPing=events
+```
+
+This should cause iOS to prompt you with a dialog asking if you want to open the URL in the Glean Sample App, and if you select "Okay" then it will launch (or resume if it's already running) the application with the indicated commands and parameters and immediately force the collection and submission of the events ping.
+
+> **Note:** This method does not work if the browser you are using to input the command is the same application you are attempting to pass the Glean debug commands to.  So, you couldn't use Firefox for iOS to trigger commands within Firefox for iOS.
```

Comment:
Uh, interesting.

Reply:
Yeah, you can't pass yourself URL's to process, apparently :/